src/js/models/menubar.js

```diff
@@ -131,88 +131,90 @@ define(function (require, exports, module) {
-                    !document.unsupported &&
-                    document.format === "Photoshop",
-                "dirty-document":
-                    (document !== null) &&
-                    document.dirty,
-                "dirty-previously-saved-document":
-                    (document !== null) &&
-                    document.dirty &&
-                    document.format,
-                "layer-selected":
-                    (document !== null) &&
-                    !document.unsupported &&
-                    (document.layers !== null) &&
-                    (document.layers.selected.size !== 0),
-                "layers-selected-2":
-                    (document !== null) &&
-                    !document.unsupported &&
-                    (document.layers !== null) &&
-                    (document.layers.selectedNormalized.size === 2),
-                "layers-selected-2+":
-                    (document !== null) &&
-                    !document.unsupported &&
-                    (document.layers !== null) &&
-                    (document.layers.selectedNormalized.size > 1),
-                "layers-selected-3+":
-                    (document !== null) &&
-                    !document.unsupported &&
-                    (document.layers !== null) &&
-                    (document.layers.selectedNormalized.size > 2),
-                "layers-selected-all-shapes":
-                    (document !== null) &&
-                    !document.unsupported &&
-                    (document.layers !== null) &&
-                    (document.layers.selected.every(function (layer) {
-                        return layer.kind === layer.layerKinds.VECTOR;
-                    })),
-                "no-background":
-                    (document !== null) &&
-                    !document.unsupported &&
-                    (document.layers !== null) &&
-                    !document.layers.backgroundSelected,
-                "no-nesting":
-                    (document !== null) &&
-                    !document.unsupported &&
-                    (document.layers !== null) &&
-                    !(document.layers.selected.some(function (layer) {
-                        return document.layers.ancestors(layer).some(function (ancestor) {
-                            return layer !== ancestor && document.layers.selected.contains(ancestor);
-                        });
-                    })),
-                "have-linked":
-                    (document !== null) &&
-                    !document.unsupported &&
-                    document.layers.hasLinkedSmartObjects,
-                "multiple-documents":
-                    Object.keys(openDocuments).length > 1,
-                "earlier-history":
-                    (document !== null) && hasPreviousHistoryState,
-                "later-history":
-                    (document !== null) && hasNextHistoryState
-            };
-        }
+        } return {
```

Comment:
Ok, I get it... but this looks really bad when it's on the same line with the ending of if